<?xml version="1.0"?>
<launch>

	<param name="use_sim_time" value="true" />
	
    <node pkg="rosbag" type="play" name="playbag" output="screen"
          args="--pause --clock --rate 1
                $(find lidar_eskf)/dat/imu_laser_2016-07-19-02-58-12_0.bag"/>
          
    	      
	<node pkg="tf" type="static_transform_publisher" name="world_view" args="0 0 0 0 0 3.1415926 world world_view 100"/> 
	
	<node pkg="lidar_eskf" type="imu_test" name="imu_test" output="screen">
	
	    <remap from="/imu"                      to="/microstrain/imu"/>
        <param name="imu_frequency"             value="50"/>
        <param name="sigma_acceleration"        value="0.1"/>
        <param name="sigma_gyroscop"            value="0.1"/>
        <param name="sigma_acceleration_bias"   value="0.0001"/>
        <param name="sigma_gyroscope_bias"      value="0.0001"/>
        <param name="gravity"                   value="9.82"/>
        
        <param name="init_bias_acc_x"           value="-0.3"/>
        <param name="init_bias_acc_y"           value="-0.1"/>
        <param name="init_bias_acc_z"           value="0.04"/>

	</node>
	
</launch>
